packages/rebound-router/lib/rebound-router.js

```diff
@@ -19,6 +19,19 @@ var SUCCESS = 'success';
 var ERROR = 'error';
 var LOADING = 'loading';
 
+// If the string passed to this function is a valid query string, split its values
+// out into a hash and return the object.
+function parseQueryString(str){
+  var res = {};
+  if(!str || str.indexOf('=') == -1) return str;
+  str = str.split('&');
+  str.forEach(function(item){
+    item = item.split('=');
+    res[item[0]] = item[1];
+  });
+  return res;
+};
```

Comment:
Mm, good catch! I'm inclined to have it add duplicates to an array and strip out trailing `[]` brackets from the keys if they exist. Sound reasonable?